### DISCUSSION TITLE
位图和字符 如何做适应布局（间距固定）

### DISCUSSION THREAD

**Discussion by 15659813252** [Tencent/libpag#2064] [Q&A] [answered]
数字777那些其实是一张张位图，位图后面是字符，如何让位图和字符之间 适应布局，也就是不要空那么大？？
<img width="375" alt="圣诞" src="https://github.com/Tencent/libpag/assets/66396822/f0548b92-e366-4829-8e8b-9ec5c50662d7">
[1.png.zip](https://github.com/Tencent/libpag/files/13840819/1.png.zip)

**Comment by kevingpqi123** [comment, marked as answer]
AE 中没有提供动态布局的相关功能，提供一种解决思路：
PAGLayer 上面有一个 setMatrix 的接口，可以修改图层的位置信息，先获取到 777 图片的位置信息，然后修改文本的位置，文本位置的修改可以参考 pag-ios 的 demo 工程

**Reply by 15659813252** [reply]
所以这是服务端的事，还是客户端的事？

**Reply by kevingpqi123** [reply]
pag 是跨平台的，你在哪个平台渲染，就在那个平台处理
